src/global_logic.js

```diff
@@ -741,6 +741,35 @@ module.exports.checkNumberofRaces = function (chara) {
     return races
 };
 
+module.exports.checkNumberOfElements = function (totals) {
+    // check num of races
```

Comment:
// check num of "elements" here.

Reply:
I am going to translate all comments. give me a few minutes and I am going to sanitize the code a little bit further.